key.go

```diff
@@ -350,6 +351,34 @@ func LoadPrivateKeyFromDER(derBlock []byte) (PrivateKey, error) {
 	return pKeyFromKey(key), nil
 }
 
+// LoadPrivateKeyByUri loads a private key similar to the openssl command.
+// For example, you can pass in "handle:0x81000000"
+//
+// This function is a more simple implementation in load_key_certs_crls that the
+// openssl command line utility uses, so most things that you can pass into the
+// command line utility should work here.
+func LoadPrivateKeyByUri(uri string) (PrivateKey, error) {
+	cstrUri := C.CString(uri)
+	defer C.free(unsafe.Pointer(cstrUri))
+
+	ctx := C.OSSL_STORE_open_ex(C.CString(uri), nil, nil, nil, nil, nil, nil, nil)
```

Comment:
`cstrUri` is unused.
Did you mean to do this?
```suggestion
	cstrUri := C.CString(uri)
	defer C.free(unsafe.Pointer(cstrUri))

	ctx := C.OSSL_STORE_open_ex(cstrUri, nil, nil, nil, nil, nil, nil, nil)
```